onnxruntime/core/providers/cuda/cu_inc/variadic_elementwise_impl.cuh

```diff
@@ -8,7 +8,7 @@ namespace cuda {
 
 template <
     typename T, typename Func,
-    size_t max_input_batch_size, size_t num_elements_per_thread>
+    int32_t max_input_batch_size, size_t num_elements_per_thread>
```

Comment:
Sorry, I should say earlier. Can you change type of num_elements_per_thread to int32 either?